doc/spec-files.rst

```diff
@@ -95,8 +95,8 @@ The statements in a spec file create instances of four classes,
 
   - ``scripts``: the python scripts named on the command line;
   - ``pure``: pure python modules needed by the scripts;
-  - ``binaries``: non-python modules needed by the scripts;
-  - ``datas``: non-binary files included in the app.
+  - ``binaries``: non-python modules needed by the scripts, including names given by the ``--add-binary`` option;
+  - ``datas``: non-binary files included in the app, including names given by the ``--add-data`` option.
```

Comment:
This section describes the `Anylysis` class, so IMHO this should be something like "names given by the ... option should be passed here". (I'm not happy with this sentence yet, though)